Automattic/wp-calypso
# Checkout: Export RadioButton from composite-checkout package

#### Changes proposed in this Pull Request

This exports the `RadioButton` component from `@automattic/composite-checkout` and uses it in new checkout to replace the radio button for plan variants.

#### Testing instructions

- Add a plan to your cart, click "Edit" on the first step, and verify that the radio buttons for the plan variants look and work as expected.
